miner/worker.go

```diff
@@ -121,7 +121,7 @@ func (env *environment) copy() *environment {
 	// to do the expensive deep copy for them.
 	cpy.txs = make([]*types.Transaction, len(env.txs))
 	copy(cpy.txs, env.txs)
-	cpy.uncles = make(map[common.Hash]*types.Header)
+	cpy.uncles = make(map[common.Hash]*types.Header, len(env.uncles))
```

Comment:
This change conflicts with https://github.com/ethereum/go-ethereum/pull/27218/files#diff-689874b30f6f905ce6c47cdefeddcf052b467a7936a981f4b6cf38939e10d924L124, where @rjl493456442 removes it. Let's revert this change and leave it be as is, it won't be present for much longer anyway